Tasks/VsTest/taskinputparser.ts

```diff
@@ -194,10 +194,20 @@ function getTiaConfiguration() : models.TiaConfiguration {
     tiaConfiguration.baseLineBuildIdFile = path.join(os.tmpdir(), uuid.v1() + '.txt');
     tiaConfiguration.useNewCollector = false;
     const useNewCollector = tl.getVariable('tia.useNewCollector');
-    if ( useNewCollector && useNewCollector.toUpperCase() === 'TRUE') {
+    if (useNewCollector && useNewCollector.toUpperCase() === 'TRUE') {
         tiaConfiguration.useNewCollector = true;
     }
-    tiaConfiguration.isPrFlow = tl.getVariable('tia.isPrFlow');
+
+    var buildReason = tl.getVariable('Build.Reason');
+
+    // https://www.visualstudio.com/en-us/docs/build/define/variables
+    if (buildReason && buildReason === "PullRequest") {
+        tiaConfiguration.isPrFlow = "true";
+    }
+    else {
+        tiaConfiguration.isPrFlow = tl.getVariable('tia.isPrFlow');
```

Comment:
If tia.isPrFlow is not defined in the task as variable, what is the default value which gets assigned here? Are we handling it in test selector?